arch/x86_64/mcount-insn.c

```diff
@@ -1,8 +1,6 @@
 #include "libmcount/internal.h"
 #include "mcount-arch.h"
 
-#define CALL_INSN_SIZE  5
-
```

Comment:
Then please move it to mcount-arch.h instead.

Reply:
this code has changed to follow your advice.